src/core/diagram/qgsstackedbardiagram.h

```diff
@@ -16,7 +16,7 @@
 #ifndef QGSSTACKEDBARDIAGRAM_H
 #define QGSSTACKEDBARDIAGRAM_H
 
-#define DIAGRAM_NAME_STACKED "Stacked"
+#define DIAGRAM_NAME_STACKED_BAR "StackedBar"
```

Comment:
I am not 100% sure, but we usually prefer static const members over #define
https://stackoverflow.com/questions/1637332/static-const-vs-define
Might be worth switching them?

Reply:
Yes, definitely worth it! Thanks for the suggestion.
I'll address this one tomorrow.

Reply:
Done in https://github.com/qgis/QGIS/pull/58568/commits/463bf249bb04c91f6cadd5cab834fd3ee836e446